Allure.Reqnroll/State/AllureReqnrollStateFacade.cs

```diff
@@ -38,6 +39,8 @@ static class AllureReqnrollStateFacade
 
     static AllureLifecycle Lifecycle { get => AllureLifecycle.Instance; }
 
+    static Dictionary<ExecutableItem, StringBuilder> OutputCache { get; } = new();
```

Comment:
I typically try to avoid static dictionaries like this, especially in parallel execution contexts like this, however i couldn't see a path forward not using something like this, without modifying the Allure.Net.Common libraries.

Reply:
That's a valid concern. Please take a look at my review comment for the alternative.